workers/analyzer/docker/Analyzer.Dockerfile

```diff
@@ -408,6 +409,25 @@ RUN mkdir -p $DOTNET_HOME/bin \
 FROM scratch AS dotnet
 COPY --from=dotnetbuild /opt/dotnet /opt/dotnet
 
+#------------------------------------------------------------------------
+# BAZEL
+FROM ort-base-image as bazelbuild
+
+ARG BAZEL_VERSION
+
+ENV BAZEL_HOME=/opt/bazel
+
+RUN mkdir -p $BAZEL_HOME/bin \
+    && if [ "$(arch)" = "aarch64" ]; then \
+    curl -L https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-linux-arm64 -o $BAZEL_HOME/bin/bazel; \
+    else \
+    curl -L https://github.com/bazelbuild/bazel/releases/download/$BAZEL_VERSION/bazel-$BAZEL_VERSION-linux-x86_64 -o $BAZEL_HOME/bin/bazel; \
+    fi \
+    && chmod a+x $BAZEL_HOME/bin/bazel
```

Comment:
@haikoschol is this still the "right" way to install Bezel, or should maybe Bazelisk be used instead by now?

Reply:
Let's go ahead with this and eventually improve later, when doing the Bazel 7.2.1 (or later) upgrade.

Reply:
This is [what Bazelisk does](https://github.com/bazelbuild/bazelisk?tab=readme-ov-file#where-does-bazelisk-get-bazel-from) essentially.

I don't think Bazelisk supports installing a specific version. The point of using it is to allow [specifying the Bazel version to run in a repository/workspace](https://github.com/bazelbuild/bazelisk?tab=readme-ov-file#how-does-bazelisk-know-which-bazel-version-to-run).